### ISSUE TITLE
Pwsh 7.1 P1 - $PSVersiontable does not mention 7.0.0 in PCompatible Versions

### ISSUE THREAD

**Issue by doctordns** [PowerShell/PowerShell#12220]
The $PSVersionTable built-invariable has a property PSCompatiibleVersions. It lists 6.0.0, 6.1.0, 6.2.0, and 7.1.0-preview.1 but NOT 7.0.0

# Steps to reproduce

```powershell
# Install 7.1.0-preview.1 - then
$PSVersionTable

```

# Expected behavior

```none
PS C:\foo> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      7.1.0-preview.1
PSEdition                      Core
GitCommitId                    7.1.0-preview.1
OS                             Microsoft Windows 10.0.19592
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0, 5.0, 5.1.10032.0, 6.0.0, 6.1.0, 6.2.0, 7.0.0, 7.1.0-preview.1}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

# Actual behavior

```none
PS C:\foo> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      7.1.0-preview.1
PSEdition                      Core
GitCommitId                    7.1.0-preview.1
OS                             Microsoft Windows 10.0.19592
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0, 5.0, 5.1.10032.0, 6.0.0, 6.1.0, 6.2.0, 7.1.0-preview.1}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

# Environment data

```none
PS C:\foo> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      7.1.0-preview.1
PSEdition                      Core
GitCommitId                    7.1.0-preview.1
OS                             Microsoft Windows 10.0.19592
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0, 5.0, 5.1.10032.0, 6.0.0, 6.1.0, 6.2.0, 7.1.0-preview.1}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by bgelens** [comment]
I added a [test](https://github.com/PowerShell/PowerShell/blob/master/test/powershell/Host/PSVersionTable.Tests.ps1#L166) to catch this. Just verified the test works locally. Wonder why it did not block the release.

**Comment by iSazonov** [comment]
Was fixed in #12184

**Comment by iSazonov** [comment]
@bgelens The test does only check a static list. You could enhance the test to check current version in the list too.
